runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/graph/LengthPrefixUnknownCodersTest.java

```diff
@@ -252,11 +255,11 @@ public void testLengthPrefixAndReplaceForRunnerNetwork() {
       }
     }
 
-    Set expectedInstructions =
-        ImmutableSet.of(
-            prefixedReadNode.getParallelInstruction(), prefixedReadNodeOut.getInstructionOutput());
-
-    assertEquals(expectedInstructions, prefixedInstructions);
+    assertThat(
+        prefixedInstructions.build(),
+        containsInAnyOrder(
+            jsonOf(prefixedReadNodeOut.getInstructionOutput()),
+            jsonOf(prefixedReadNode.getParallelInstruction())));
```

Comment:
Created `GenericJsonMatcher.jsonOf`, to leverage Hamcrest's `containsInAnyOrder`.